### PR TITLE
Docstrings

### PR DESCRIPTION
**Pull request by mzouink**

### CODE REVIEW

**Comment by cmalinmayor** [comment]
Need to manually add each documented class to the toctree in `/docs/source/api/....`.
Can build docs locally (after installing `pip install -e .[dev, docs]` and/or isntalling sphinx and sphinx material theme) by running `sphinx-build docs/source docs/dist` and then view the website by opening `docs/dist/index.html`

**Review by rhoadesScholar — DISMISSED** [review]

**Comment by mzouink** [comment]
Ready to be checked !

**Comment by mzouink** [comment]
@rhoadesScholar  is not draft anymore

File: dacapo/experiments/tasks/predictors/hot_distance_predictor.py
Comment (cmalinmayor) [inline dacapo/experiments/tasks/predictors/hot_distance_predictor.py:1]: This file got messed up. Need to fix @mzouink

Comment (mzouink) [reply dacapo/experiments/tasks/predictors/hot_distance_predictor.py:1]: fixed the file

File: dacapo/experiments/tasks/evaluators/dummy_evaluation_scores.py
Comment (cmalinmayor) [inline dacapo/experiments/tasks/evaluators/dummy_evaluation_scores.py:1]: This one is also messed up, docstrings can't go between the decorator and the function. Maybe check that the code runs/the docs build before we review. (I'm finding these by a fresh pull and building the docs). @mzouink

Comment (mzouink) [reply dacapo/experiments/tasks/evaluators/dummy_evaluation_scores.py:1]: i will do a second checking round

**Review by rhoadesScholar — APPROVED** [review]

**Pull request merged**